diagrams/server.drawio

```diff
@@ -1,112 +1 @@
-                </mxCell>
-                <mxCell id="Rp-ab2z_M1Jg2SWGDdXT-57" value="" style="edgeStyle=orthogonalEdgeStyle;rounded=0;orthogonalLoop=1;jettySize=auto;html=1;fontSize=16;" parent="1" source="Rp-ab2z_M1Jg2SWGDdXT-53" target="Rp-ab2z_M1Jg2SWGDdXT-44" edge="1">
-                    <mxGeometry relative="1" as="geometry"/>
-                </mxCell>
-                <mxCell id="Rp-ab2z_M1Jg2SWGDdXT-53" value="&lt;h6&gt;&lt;font style=&quot;font-size: 16px&quot;&gt;&lt;span style=&quot;font-weight: normal&quot;&gt;CLI Tool&lt;/span&gt;&lt;/font&gt;&lt;/h6&gt;" style="shape=process;whiteSpace=wrap;html=1;backgroundOutline=1;sketch=0;fontSize=30;fillColor=#f8cecc;strokeColor=#b85450;" parent="1" vertex="1">
-                    <mxGeometry x="140" y="100" width="110" height="30" as="geometry"/>
-                </mxCell>
-            </root>
-        </mxGraphModel>
-    </diagram>
-</mxfile>
+<mxfile host="app.diagrams.net" modified="2022-11-15T17:21:21.538Z" agent="5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36" etag="BEC-PMrsOuAj8PANLo-K" version="20.1.1" type="device"><diagram id="TgZHtt84LtUgyle7RrUV" name="Server">7VrbdqM2FP0aP6YLcTN+dJw0M23STuvpysxTl4BjoMGICjm28/UVIMxFxMZj7MGz+mR0JGTYZ5+rGGmz5eaB4th/Ii6EI1VxNyPtbqSqCFkm/0kl21wyNvVc4NHAFYtKwTx4AyFUhHQVuJDUFjJCQhbEdaFDoggcVpNhSsm6vmxBwvq/xtgDSTB3cChLnwOX+bnUMpRS/gECzy/+GSliZomLxUKQ+Ngl64pIux9pM0oIy6+WmxmEKXgFLvl9P78zu3swChHrcsOf8Q221be/n9Avnjp/frhzv3y+Ebu84nAlXlg8LNsWCFCyilxIN1FG2u3aDxjMY+yks2uucy7z2TLkI8QvF0EYzkhIaHavtlgsVMfh8oRR8gKVGde0TcPkM/JrFM8ElMGmIhKv9QBkCYxu+ZJNnSuCYrvxulTYuFCLX1GWVgixIIm327rEkV8IKI+AVZtIMILLeSWGhDKfeCTC4X0pva0DXa55JCQW8P4DjG2FkeAVI3Xw34UyISvqwJ7nFVpnmHrA9r2XWJi+zF7NUAgxC17rRtQ7zGoLe82Qv8CtG7zySy+9vMMMzxmhUMzxv6pM7+E7OprvgFwDxm18n5hjDffEd73Od02T+Y7a+X4uPbQp4pr4r3XkP5oMiv9aF/7/CtsnHPE4RX8YA0DjoVmAhiQYB014vavDVwZFeF0i/F8xZxrg5XTF0aMB214hnXd5yD46q5ekMxp/DzrDJmBf0tt/MsToa2XmbiN2zgZbMfh2EzA6moA1KAtA1nU5mq4oIzQomA3J0fzGS9spY5DwVPIKfYyhDi1kFlns1SaNVkdqq9qgqG1JsE+f570ymvPZcvU2RluqrZk9MdpsRE1dlRmtXzQH1K8laHLQ6Ta/SS+GX7MhKobljdmoducnoAGHDOjJMbjw+odtSO/bhrJbp5TibWVBTIKIJRITdvufELflTtv0bZU2Jfo0PAzWorXdZjoW2Iue2g+Tw4ZnXDRbnUjYLmFJ6Im1QPICzPEFxxtI28h1F0ob0kgZaxPoqc7tgLR50UaP3HFoCx58pyBOUhASH8ep0AnJyj0N8stEFaPRWjD0jnmSfjbM5aK33XOcB/XLuJSxMTDUNTk7nU0vA7mtgAZmG+QKWIpl9dR0aJyaGF0LgvNBrkqQt7QzC5FNm5JmW7PPW+d/PHZemx442DiB7n/XYNWOS9sw4OGKqof5ZOeB7dHeCbDz4mXh7vcV49tAQbw8j0TGIAKceZiDbfHNOltNKvvaQme+Wuhr+uljRZGlfE+20eHkdJ8yFtgd41YnbOumoeg9ZRvfeIx6Noegm5I25kBfQe6/8DdkDThrSEUkNYAarEKEw8CLUlvjuKUFzW2KV+DgcComloHrZpVam/7qGq6rkERMFG3abiyeuOXs/nj/3TgFVMfdyl/1XNoy5J6ZHC/PXw9XcEd7c8LDZySirDhYnOq9F6enmY1cDu2cmFnTh/nvKv36JMPsJo8MPMdRkBlvyslm8EoXn75LEuPo/V3WgrDpPhGhSxzKO2VuWJnhMEwq7jjfVwq3+UNL4hSPA0E4psSBJOkQgt+LuHu9QsPRWw60fy9jW4Zu7G22HHPeVM+2d2leNfVDl6wrDdnTX7fvMLo2tgbmOwy5TdWS4U49yNpkR2WzZzOkTFXfxZD0RrNgAIY0/tEMqeunOUMzpPe/zakGnWGF1NljGlI/ExL+H1GPK530yzkCPiw/Gs6PJ8pPr7X7/wA=</diagram></mxfile>
```

Comment:
This change from an XML file format to a uuencoded file format seems like a slight security risk since the uuencoded version is more obfuscated. However, I'm not a security professional. Just flagging it here for consideration.

Reply:
I think this may be a change a change that happened over time with the Drawio / DiagramsNet product. (The original diagram was exported by Draw.io . The current app, rebranded as Diagrams.Net, can read / use both files.)

Reply:
I double checked this value by loading in the old XML into draw io and then re-exporting, which produced a match on the updated encoding

Reply:
It's now an XML file :)

Reply:
thank you 🙏 it all looks good to me now, it can be merged as soon as the DCO is fixed